src/test/java/com/bobocode/session/SessionExceptionsTest.java

```diff
@@ -0,0 +1,96 @@
+package com.bobocode.session;
+
+import com.bobocode.Dialect;
+import com.bobocode.exception.EntityMappingException;
+import com.bobocode.session.dto.EntityClass;
+import com.bobocode.session.dto.NotDefinedIdField;
+import com.bobocode.session.dto.NotEntityClass;
+import org.assertj.core.api.ThrowableAssert;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+import javax.sql.DataSource;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.when;
+
+class SessionExceptionsTest {
+
+    @Mock
```

Comment:
Hm, don't those annotations require you to put MockitoExtension on top of test class?

Reply:
Yes, you're right. I didn't know about it.
Here is good [article](https://mincong.io/2020/04/19/mockito-junit5/#mockitoextension). And I checked documentation as well.

Reply:
done